### ISSUE TITLE
[Request] flowblade-git

### ISSUE THREAD

**Issue by tio-trom** [chaotic-aur/packages#1766]
### Link to the package(s) in the AUR

https://aur.archlinux.org/packages/flowblade-git

### Utility this package has for you

multitrack non-linear video editor for Linux - I edit many hours of videos and this one looks very promising!

### Do you consider the package(s) to be useful for every Chaotic-AUR user?

YES!

### Do you consider the package to be useful for feature testing/preview?

- [ ] Yes

### Have you tested if the package builds in a clean chroot?

- [X] Yes

### Does the package's license allow redistributing it?

YES!

### Have you searched the issues to ensure this request is unique?

- [X] YES!

### Have you read the README to ensure this package is not banned?

- [X] YES!

**Comment by Technetium1** [comment]
dependencies require python2, so this will be removed.

**Comment by tio-trom** [comment]
Yeah as far as I understand `python2` is outdated now and no more security fixes. Makes sense . Hopefully flowblade will change dependencies for arch. https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/DEPENDENCIES.md

**Comment by Technetium1** [comment]
Done
